### ISSUE TITLE
TypeError: send_command() got an unexpected keyword argument 'use_ttp'

### ISSUE THREAD

**Issue by musamakhalid88** [ktbyers/netmiko#2232]
I am using ttp parser but it is giving me this error. below is the code please suggest solution.

```python
from netmiko import ConnectHandler
from getpass import getpass
import time


password = getpass()
time.sleep(4)

#for ssh
huawei_SW2 = {
    'device_type': 'huawei',
    'ip': '192.168.1.4',
    'username': 'netadmin',
    'password': password,
    'session_log' : "AR1.txt"
}

# write template to file
ttp_raw_template = """
interface {{ interface }}
 description {{ description }}
"""

all_devices = [huawei_SW2]

for devices in all_devices:
    
    print("SSH to " ,devices["ip"])
    net_connect = ConnectHandler(**devices)
    print(net_connect.find_prompt())
    output = net_connect.send_command('disp current-configuration interface', use_ttp=True, ttp_template= ttp_raw_template)
    print(output)
    net_connect.disconnect()
```

**Comment by ktbyers** [comment]
Which version of Netmiko are you using?

**Comment by musamakhalid88** [comment]
below is the current version.

Requirement already satisfied: scp>=0.13.2 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from netmiko) (0.13.2)
Requirement already satisfied: setuptools>=38.4.0 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from netmiko) (41.2.0)
Requirement already satisfied: paramiko>=2.4.3 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from netmiko) (2.7.1)
Requirement already satisfied: textfsm in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from netmiko) (1.1.0)
Requirement already satisfied: pyserial in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from netmiko) (3.4)
Requirement already satisfied: cryptography>=2.5 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from paramiko>=2.4.3->netmiko) (2.9.2)
Requirement already satisfied: bcrypt>=3.1.3 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from paramiko>=2.4.3->netmiko) (3.1.7)
Requirement already satisfied: pynacl>=1.0.1 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from paramiko>=2.4.3->netmiko) (1.3.0)
Requirement already satisfied: cffi>=1.1 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from bcrypt>=3.1.3->paramiko>=2.4.3->netmiko) (1.14.0)
Requirement already satisfied: six>=1.4.1 in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from bcrypt>=3.1.3->paramiko>=2.4.3->netmiko) (1.15.0)
Requirement already satisfied: pycparser in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from cffi>=1.1->bcrypt>=3.1.3->paramiko>=2.4.3->netmiko) (2.20)
Requirement already satisfied: future in c:\users\m.usama\appdata\local\programs\python\python38-32\lib\site-packages (from textfsm->netmiko) (0.18.2)


napalm                 2.5.0
napalm-huawei-vrp      0.1.5
ncclient               0.6.9
netaddr                0.8.0
netmiko                2.4.2
ntc-templates          1.5.0
nxapi-plumbing         0.5.2
paramiko               2.7.1

**Comment by ktbyers** [comment]
`netmiko 2.4.2`

use_ttp requires a newer version of Netmiko than that. I would recommend that you upgrade to Netmiko 3.4.0.


Regards, Kirk

**Comment by musamakhalid88** [comment]
Thank you sir for your kind support my issue is resolved after upgrade to netmiko 3.4.0

output of ttp :

SSH to  192.168.1.4
<R1>
[[[{'interface': 'Vlanif10'},
   {'interface': 'Ethernet0/0/0'},
   {'interface': 'Ethernet0/0/1'},
   {'interface': 'Ethernet0/0/2'},
   {'interface': 'Ethernet0/0/3'},
   {'interface': 'Ethernet0/0/4'},
   {'interface': 'Ethernet0/0/5'},
   {'interface': 'Ethernet0/0/6'},
   {'interface': 'Ethernet0/0/7'},
   {'interface': 'Serial2/0/0'},
   {'interface': 'Serial2/0/1'},
   {'interface': 'GigabitEthernet0/0/0'},
   {'interface': 'GigabitEthernet0/0/1'},
   {'interface': 'NULL0'},
   {'interface': 'LoopBack0'},
   {'interface': 'LoopBack50'}]]]
